aries_vcx/src/common/ledger/service_didsov.rs

```diff
@@ -1,14 +1,30 @@
 pub const SERVICE_SUFFIX: &str = "indy";
+
 pub const SERVICE_TYPE: &str = "IndyAgent";
 
-// Service object as defined https://github.com/hyperledger/aries-rfcs/blob/main/features/0434-outofband/README.md#the-services-item
-// Note that is divergence from w3c spec https://w3c.github.io/did-core/#service-properties
-#[derive(Debug, Deserialize, Serialize, Clone,PartialEq)]
+// https://sovrin-foundation.github.io/sovrin/spec/did-method-spec-template.html
+#[derive(Debug, Deserialize, Serialize, Clone)]
+#[cfg_attr(test, derive(PartialEq))]
 pub struct EndpointDidSov {
     pub endpoint: String,
     #[serde(default)]
-    #[serde(rename = "routingKeys")]
+    #[serde(rename = "camelCase")]
```

Comment:
will this behave correctly? i.e. will it be literally `"camelCase"`

Reply:
uhh this was great catch, signifying importance of de/serialization tests - these has now been added